Description: adjust "native.cgroupdriver" default value to always be "cgroupfs" irrespective of systemd since Docker cannot handle the new unified hierarchy
Author: Tianon Gravi <tianon@debian.org>
Bug: https://github.com/docker/docker/issues/16256
Bug-Debian: https://bugs.debian.org/798778

diff --git a/daemon/execdriver/native/driver.go b/daemon/execdriver/native/driver.go
index 94f200a..b09c86c 100644
--- a/daemon/execdriver/native/driver.go
+++ b/daemon/execdriver/native/driver.go
@@ -74,9 +74,9 @@ func NewDriver(root, initPath string, options []string) (*Driver, error) {
 	// this makes sure there are no breaking changes to people
 	// who upgrade from versions without native.cgroupdriver opt
 	cgm := libcontainer.Cgroupfs
-	if systemd.UseSystemd() {
-		cgm = libcontainer.SystemdCgroups
-	}
+	//if systemd.UseSystemd() {
+	//	cgm = libcontainer.SystemdCgroups
+	//}
 
 	// parse the options
 	for _, option := range options {
